src/AbstractRoboFile.php

```diff
@@ -1315,6 +1317,30 @@ protected function getLocalSettings($app = null, $timestamp = null)
         return $this->tokenReplace($this->getConfig()->get('local'), $replacements) + $defaults;
     }
 
+    protected function getTimeoutSetting($setting = null)
```

Comment:
This needs documentation on how to overrule this.

Reply:
as in, how to overrule in the property files? Or as in http://php.net/manual/en/language.oop5.inheritance.php

Reply:
First one :)

Reply:
Added a docblock. Real documentation should probably get fixed in #3 someday...